client/signup/config/steps-pure.js

```diff
@@ -349,6 +346,21 @@ export function generateSteps( {
 			dependencies: [ 'siteSlug', 'domainItem' ],
 			providesDependencies: [ 'cartItem' ],
 		},
+
+		'mailbox-plan': {
+			stepName: 'mailbox-plan',
+			apiRequestFunction: addPlanToCart,
+			fulfilledStepCallback: isPlanFulfilled,
+			dependencies: [ 'siteSlug', 'emailItem' ],
+			providesDependencies: [ 'cartItem' ],
+			props: {
+				get headerText() {
+					return i18n.translate( 'Want a website to expand your online presence?' );
```

Comment:
This was what we originally had in the requirements, but I'm having seconds thoughts about the wording...

> Want a website to establish your online presence?

Which of `establish` and `expand` sound better?

@semihakocer Any recommendation here?

Reply:
Both might work. We use establish in a few different places IIRC

Reply:
Thanks @semihakocer In that case, @fredrikekelund let's use `establish` to be consistent!